src/Transports.Subscriptions.WebSockets/WebSocketConnection.cs

```diff
@@ -26,6 +26,7 @@ public virtual async Task Connect()
 
         public void Dispose()
         {
+            _server.Dispose();
```

Comment:
Does `WebSocketConnection` owns `SubscriptionServer` ? Are you sure?

Reply:
Yes. Check the factory. It’s created by the factory and passed to the connection. Perhaps a poor design but that’s what it is in this version.

Reply:
(On the road now so I’m going from memory…)

Reply:
Double checked: yes it does.

Reply:
OK